internal/sdkv2provider/schema_cloudflare_zone_settings_override.go

```diff
@@ -567,6 +567,26 @@ var resourceCloudflareZoneSettingsSchema = map[string]*schema.Schema{
 			},
 		},
 	},
+
+	"aegis": {
+		Type:     schema.TypeList,
+		Optional: true,
+		Computed: true,
+		MinItems: 1,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"enabled": {
+					Type:     schema.TypeBool,
+					Optional: true,
```

Comment:
you can add `Description` fields here to have it automatically generate in the documentation as well.

Reply:
done